CMakeLists.txt

```diff
@@ -97,6 +98,17 @@ if(FREESCALE_FOUND)
   install(TARGETS moonlight-imx DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
+if(ROCKCHIP_FOUND)
+  list(APPEND MOONLIGHT_DEFINITIONS HAVE_ROCKCHIP)
+  list(APPEND MOONLIGHT_OPTIONS ROCKCHIP)
+  set(CMAKE_C_FLAGS_DEBUG "-D_DEBUG")
```

Comment:
in case ``cmake -DCMAKE_BUILD_TYPE=Debug .`` is specified it would add _DEBUG to the CFLAGS, would it be ok to ``#ifdef _DEBUG`` all console output ?